rest-service/manager_rest/test/endpoints/test_deployment_updates.py

```diff
@@ -123,6 +124,89 @@ def test_remove_node_and_relationship(self):
             self.assertEquals(1 if param in changed_params else 0,
                               len(execution.parameters[param]))
 
+    def test_plugin_installation_updates_plugins_by_default(self):
+        def assert_all_not_in(plugin_names, plugins_set):
+            for plugin_name in plugin_names:
+                self.assertNotIn(plugin_name, plugins_set)
+
+        def assert_all_in(plugin_names, plugins_set):
+            for plugin_name in plugin_names:
+                self.assertIn(plugin_name, plugins_set)
+
+        deployment_id = 'dep'
+        changed_params = ['update_plugins',
+                          'central_plugins_to_install',
+                          'central_plugins_to_uninstall']
+
+        self._deploy_base(deployment_id, 'setup_with_plugin_1.yaml',
+                          skip_plugins_validation=True)
+
+        self._update(deployment_id, 'setup_with_plugin_2.yaml')
+        dep_update = \
+            self.client.deployment_updates.list(deployment_id=deployment_id)[0]
+
+        execution = self.client.executions.get(dep_update.execution_id)
+        exec_params = execution.parameters
+        for changed_param in changed_params:
```

Comment:
There is an assert that asserts lists without order

Reply:
Same as #discussion_r293823307